src/de/dkfz/b080/co/aceseq/ACESeqWorkflow.java

```diff
@@ -20,28 +20,37 @@
  */
 public class ACESeqWorkflow extends WorkflowUsingMergedBams {
 
+
+    private boolean allowMissingSVFile(ExecutionContext context) {
+        return getflag(context, "allowMissingSVFile", false);
+    }
+
+    private boolean runWithSV(ExecutionContext context) {
+        return getflag(context, "runWithSv", true);
+    }
+
     @Override
     public boolean execute(ExecutionContext context, BasicBamFile _bamControlMerged, BasicBamFile _bamTumorMerged) {
 
         BamFile bamControlMerged = new BamFile(_bamControlMerged);
         BamFile bamTumorMerged = new BamFile(_bamTumorMerged);
 
-        boolean runWithSv = context.getConfiguration().getConfigurationValues().getBoolean("runWithSv", true);
-        boolean runWithCrest = context.getConfiguration().getConfigurationValues().getBoolean("runWithCrest", false);
-        boolean runQualityCheckOnly = context.getConfiguration().getConfigurationValues().getBoolean("runQualityCheckOnly", false);
-        boolean runWithFakeControl = context.getConfiguration().getConfigurationValues().getBoolean("runWithFakeControl", false);
-        boolean runWithoutControl = context.getConfiguration().getConfigurationValues().getBoolean("runWithoutControl", false);
+        boolean runWithSV = runWithSV(context);
```

Comment:
Good to have this repetitive code simplified, and it's kind of o.k. to do that with functions. But you are aware that in the AQCWF workflows we use a configuration object that gives a really neat syntax `cfg.propertyName` for accessing configuration values. Would it make sense to use something similar here?

Reply:
Depends. We can do that. However, I wanted to concentrate an simple changes. It might make sense.